File: thorium/llmGenerated/testFilter.R
csv <- "data.csv"
table <- read.csv(csv, stringsAsFactors = FALSE)
table <- subset(table, department == "INFO" & age >= 20)
table <- subset(table, department == "GPM")
table <- subset(table, city == "Lyon" & age >= 22)
write.csv(table, "data2.csv", row.names=FALSE, quote=FALSE)

File: thorium/llmGenerated/testFilter.py
import pandas as pd

csv = "data.csv"
table = pd.read_csv(csv)

table = table[(table['department'] == "INFO") & (table['age'] >= 20)]

table = table[table['department'] == "GPM"]

table = table[(table['city'] == "Lyon") & (table['age'] >= 22)]

table.to_csv("data2.csv",index=False)